### PR TITLE
Docker + auto-building + live-reload

### PR DESCRIPTION
**Pull request by jamespacileo**
I've tested these and it seems to be working alright, would be great if someone else could run it just to double check before we merge this in.

Also added line endings for the Posix Nazi 😄 strange it's not enabled by default

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -25,3 +25,7 @@ defaults:
         type: meetups
     values:
         layout: meetup
+
+# deployment
+host: 0.0.0.0
+port: 4000
```
Comment (adamchainz) [inline _config.yml]: you missed a bit

**Comment by jamespacileo** [comment]
little change added

File: _config.yml (hunk 1 above)
Comment (jamespacileo) [inline _config.yml]: 👍

**Comment by seddonym** [comment]
This is working for me on Linux - happy to merge, unless Adam you want to test too?

**Comment by adamchainz** [comment]
Merge away

**Pull request merged**